project/password/templates/registration/password_change.html

```diff
@@ -0,0 +1,52 @@
+{% extends 'base.html' %}
+
+{% block content %}
+<style type="text/css">
+  .form-signin {
+    width: 100%;
+    max-width: 330px;
+    padding: 15px;
+    margin: auto;
+  }
+  .form-signin .checkbox {
+    font-weight: 400;
+  }
+  .form-signin .form-control {
+    position: relative;
+    box-sizing: border-box;
+    height: auto;
+    padding: 10px;
+    font-size: 16px;
+  }
+  .form-signin .form-control:focus {
+    z-index: 2;
+  }
+  .form-signin input[type="password"] {
+    margin-bottom: 10px;
+    border-bottom-right-radius: 0;
+    border-bottom-left-radius: 0;
+  }
+  .h3{
+    text-align: center;
+  }
+</style>
+
+
+<form method="POST" class="form-signin">{% csrf_token %}
+  <h1 class="h3 mb-3 font-weight-normal">Change password</h1>
```

Comment:
Add i18n tags here so we can translate this text.
```suggestion
  <h1 class="h3 mb-3 font-weight-normal">
      {% translate "Change password" %}
  </h1>
```